pages/about.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- Start of Google Fonts and Icons -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined" rel="stylesheet">
+  <!-- End of Google Fonts and Icons -->
+  <link href="../styles/template.css" rel="stylesheet"/>
+  <link href="../styles/facts.css" rel="stylesheet"/>
+  <title>About</title>
+  
+</head>
+<body>
+<nav id="standard-nav">
+  <ul class="poppins-regular">
+    <li id="city-title">AUSTIN, TX</li>
+    <li class="nav-links">
+      <a href="../index.html">Home</a>
+    </li>
+    <li class="nav-links">
+      <a href="./gallery.html">Gallery</a>
+    </li>
+    <li class="nav-links">
+      <a href="./facts.html">Fun Facts</a>
+    </li>
+    <li class="nav-links">
+      <a href="./about.html">About</a>
+    </li>
+  </ul>
+  <div id="color_banner">
+    <div style="background:#BF5700;">&nbsp;</div>
+    <div style="background:#236B6B;">&nbsp;</div>
+    <div style="background:#D6A7A1;">&nbsp;</div>
+    <div style="background:#DFAF2B;">&nbsp;</div>
+    <div style="background:#4A7C9E;">&nbsp;</div>
+  </div>
```

Comment:
Looking at this again, I wonder if this color banner should be included in the `nav` tag or should it be moved outside of it and then grouped together with something like a `div` tag?